src/main/java/com/_up/megastore/services/implementations/ProductService.java

```diff
@@ -69,20 +75,22 @@ private void ifProductIsNotDeletedThrowException(Product product){
         }
     }
 
-    private String saveProductImage(MultipartFile multipartFile) {
-        return fileUploadService.uploadImage(multipartFile);
+    private List<ProductImage> saveProductImages(MultipartFile[] multipartFiles) {
+        return Arrays.stream(multipartFiles)
+                .map(productImageService::saveProductImage)
+                .collect(Collectors.toList());
```

Comment:
Quizás esto se lo podriamos delegar al ProductImageService, generamos un metodo en el servicio "saveProductImages", le pasamos la lista de multipartFiles y que sea este el que recorra cada elemento para guardar la imagen, what do you think?

Reply:
Me gusta